packages/validator-rules/lib/AmpValidatorRules.js

```diff
@@ -158,15 +158,21 @@ class AmpValidatorRules {
         .filter((tag) => !tag.extensionSpec)
         .map((tag) => {
           tag.attrs = tag.attrs || [];
+
+          // Merge global attribute lists into atrrs
```

Comment:
nit: s/atrrs/attrLists

Your comment explains what you're doing, but I get that from the code. I'm more interested in the why (which I don't understand)

Reply:
Tried rewriting. The key thing is that `attrLists` contains a list of IDs that are looked up from the top-level attrLists property. It doesn't contain any *real* attribute definition. This bit of code merges the real attribute definitions (from the global object) into `attrs`.

Let me know if the current comment makes sense to convey that.